packages/react/src/runtimes/local/LocalThreadRuntimeCore.tsx

```
@@ -268,7 +287,12 @@ export class LocalThreadRuntimeCore
     try {
       this._lastRunConfig = runConfig ?? {};
       const context = this.getModelContext();
-      const promiseOrGenerator = this.adapters.chatModel.run({
+
+      runCallback =
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)